coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,9 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter, available_filters):
```

Comment:
Please support `None` values for those. And if one of those parameters is `None`, it shall not be printed.